package-lock.json

```diff
@@ -4,6 +4,11 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@adobe/css-tools": {
+      "version": "4.0.1",
+      "resolved": "https://artifactory.corp.adobe.com/artifactory/api/npm/npm-adobe-release/@adobe/css-tools/-/css-tools-4.0.1.tgz",
```

Comment:
@holblin Adobe private npm registry ? We should use `registry.npmjs.org`, can you fix it ?  CI failed [here](https://github.com/stylus/stylus/runs/7701227580?check_suite_focus=true#step:4:13) . Btw, thanks for your contribution.

Reply:
Yes 👍 I need to redo the installation. I'll do that Monday ;-)